lib/pushmi_pullyu/aip/creator.rb

```diff
@@ -0,0 +1,66 @@
+require 'archive/tar/minitar'
+require 'bagit'
+require 'pushmi_pullyu/aip'
+
+class PushmiPullyu::AIP::Creator
+
+  attr_reader :aip_filename, :aip_directory
+
+  def initialize(noid)
+    @noid = noid
+    @aip_directory = PushmiPullyu::AIP.aip_directory(noid)
+    @aip_filename = PushmiPullyu::AIP.aip_filename(noid)
+    PushmiPullyu::AIP.validate(noid)
+  end
+
+  def run(should_skip_download: false, should_clean_work_directories: true)
+    download_aip unless should_skip_download
```

Comment:
This is done: the downloaded and the creator are decoupled and the 'should_skip_download' option is removed (I know, I said I wouldn't work on this any more this sprint, but this didn't take very long).

I also added some test fixture data to mock previously downloaded AIP data for the creator tests to work on.